org-cyf-guides/content/pair-programming/_index.md

```diff
@@ -0,0 +1,19 @@
++++
+title = 'Pair programming'
+description = 'Practice writing code and getting support'
+emoji = '👥'
++++
+
+Pair programming is an excellent way to develop programming and communication skills.
+
+It is often much easier to work through something when working on something 1-to-1.
```

Comment:
Minor: Maybe this could be something like "It is often much easier to work through a problem when working with someone 1-to-1." ?